docs/modelserving/v1beta1/transformer/collocation/README.md

```diff
@@ -143,3 +152,240 @@ curl -v -H "Host: ${SERVICE_HOSTNAME}" -H "Content-Type: application/json" -d $I
     * Connection #0 to host localhost left intact
     {"predictions":[2]}
     ```
+
+## Collocation with Runtime
+### Deploy the InferenceService
+
+Since, the predictor and the transformer are in the same pod, they need to listen on different ports to avoid conflict. `Transformer` is configured to listen on port 8080 (REST) and 8081 (GRPC) 
+while, `Predictor` listens on port 8085 (REST). `Transformer` calls `Predictor` on port 8085 via local socket. 
```

Comment:
Maybe also mention which is the default port for gRPC for predictors.

Reply:
it differs for different serving runtimes, can be documented on serving runtime page.